### PR TITLE
#165674780 fix bugs on backend

### PR DESCRIPTION
**Pull request by geekdanitek**
#### Type of change
- [x] Bug fix (none breaking change which fixes an issue)
#### Story (PT)
[Delivers #165674780](https://www.pivotaltracker.com/story/show/165674780)

### CODE REVIEW

**Comment by Chiazokam** [comment]
Good job. But travis seems to be failing

**Review by TheDrizzyWay — APPROVED** [review]
Good job. What's up with travis though?

File: server/utils/comments/comments.util.js [hunk 1]
```diff
@@ -42,11 +42,10 @@ class commentValidation {
 
     const userComment = await Comment.findOne({
       where: {
-        userId,
         id
       }
     });
-    if (!userComment) {
+    if (userId !== userComment.userId) {
       return Response(res, 400, 'Only the user of this comment can edit comment');
```
Comment (samorahno) [inline server/utils/comments/comments.util.js]: Good job @geekdanitek, Kindly correct this response message and make it more descriptive. You might as well look into the test cases to correct this if it exists there.

Comment (geekdanitek) [reply server/utils/comments/comments.util.js]: Ok @samorahno this will be implemented.

**Review by chukwudidu — APPROVED** [review]
Good job guys. We did a good job.

**Comment by BukkyOmo** [comment]
Good

**Review by davidokonji — APPROVED** [review]
Good job done guys

**Comment by Chiazokam** [comment]
> Good job. But travis seems to be failing

I guess it's a travis issue

**Review by Chiazokam — APPROVED** [review]

**Review by samorahno — APPROVED** [review]
Looks good to me

File: server/utils/comments/comments.util.js (hunk 1 above)
Comment (geekdanitek) [reply server/utils/comments/comments.util.js]: @samorahno Feedback implemented.

**Comment by chukwudidu** [comment]
> > Good job. But travis seems to be failing
> 
> I guess it's a travis issue

Yes it was. For some weird reason, Travis tried using nodejs version 12 to run the tests so @geekdanitek and I specified version 11 in the `.travis.yml` config after looking through the previous Travis builds.

**Review by BukkyOmo — APPROVED** [review]

**Pull request merged**